src/components/field/index.js

```diff
@@ -13,17 +13,20 @@ import styles from './styles'
 import TextInputMask from 'react-native-text-input-mask'
 
 function startAnimation(animation, options, callback) {
+
```

Comment:
As a best practices thing, it's good to avoid extra whitespace changes since they make it harder to skim a code review. (Don't worry about it here, just something for the future. I assume `prettier` just went nuts with this file.)

Reply:
Prettier I don't think is complaining about the file, will see if I can figure out enabling prettier for this file because I don't want white spaces everywhere in the file.